### PR TITLE
[ENH] Keywords - replace embedding with MBERT

### PR DESCRIPTION
**Pull request by PrimozGodec**
##### Issue

Embedding keyword has poor performance and is slow.

##### Description of changes
Replace Embedding keyword extractor with MBERT with excellent performance. This keyword extractor extracts keywords with higher F1 scores than all existing extractors. The downside of this extractor is that it, on average extracts less than 5 keywords per document and also 0 many times.

##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

**Review by VesnaT — APPROVED** [review]

**Comment by VesnaT** [comment]
Event with the two comments I'm happy to merge.

File: orangecontrib/text/widgets/utils/__init__.py [hunk 1]
```diff
@@ -1,3 +1,24 @@
+from enum import IntEnum, Enum
+from typing import Union
+
 from .decorators import *
 from .widgets import *
 from .concurrent import asynchronous
+
+
+def enum2int(enum: Union[Enum, IntEnum]) -> int:
```
[the remark this answers is not in the document]
Comment (PrimozGodec) [reply orangecontrib/text/widgets/utils/__init__.py]: This one is actually already merged. I forgot to rebase this PR. :) Anyway enum2int from Orange is not released yet. I will add a test that will start to fail after some time and warn about removing it from text after some time.

[the remark this answers is not in the document]
Comment (PrimozGodec) [reply orangecontrib/text/widgets/utils/__init__.py]: Actually it was written here before it existed in Orange. I propose a time bomb that will start to fail and warn about removing it. https://github.com/biolab/orange3-text/pull/951

**Comment by PrimozGodec** [comment]
@VesnaT since you approved it I am merging it.

**Pull request merged**